doc/source/example/reference/commands/select/drilldowns_label_table.log

```diff
@@ -0,0 +1,140 @@
+Execution example::
+
+  table_create Tags TABLE_PAT_KEY ShortText
+  # [
+  #   [
+  #     -22, 
```

Comment:
Is this expected?

Reply:
I am trying to resolve this...
Maybe this is because another example in select.rst already creates `Tags` table...

Reply:
I have renamed `Tags` to `NestedDrilldownTags` and `Memos` to `NestedDrilldownMemos`.